code-challenges.test.js

```diff
@@ -12,20 +12,67 @@
 
 // --------------------1) Create a function that takes in a number (greater than 2) and returns an array that length containing the numbers of the Fibonacci sequence.
 
-// a) Create a test with expect statements for each of the variables provided.
+                  ///First and foremost I begin to set up my TDD for jest with "describe" and "it" statments. |sequenceNums| will be the function used later.
+                  ///The "it" statement will return the correct numbers of the Fibonacci sequence.
+                  ///I was required to look at the numbers right off the bat to figure out what this sequence was all about, I discovered the numbers were the sum of the 
+                  ///previous one. In this case, every element in the arrays.
+                  ///I was able to get a solid failure stating: ReferenceError: sequenceNums is not defined. Next I construct my function.
+                  /// I did some research to shine a light on what iteration was needed for this problem, and most websites were pointing to for loops, 
+                  ///which I had suspected.
+                  /// I'm trying to set up the for loop to iterate over each element and add the previous element. 
 
+// a) Create a test with expect statements for each of the variables provided.
 const fibLength1 = 6
 // Expected output: [1, 1, 2, 3, 5, 8]
 
 const fibLength2 = 10
 // Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
+     
+
+describe("sequenceNums",() => {
+        it ("returns an array that length containing the numbers of the Fibonacci sequence", () =>{
+          expect(sequenceNums(fibLength1)).toEqual([1, 1, 2, 3, 5, 8])
+          expect(sequenceNums(fibLength2)).toEqual([1, 1, 2, 3, 5, 8, 13, 21, 34, 55])
+        })
+      })
+                  // TEST RESULTS: Failed        ReferenceError: sequenceNums is not defined
+                  //           Test Suites: 1 failed, 1 total
+                  //           Tests:       1 failed, 1 total
+
 
 // b) Create the function that makes the test pass.
 
+    
+let fibArr = (array) => {
+  for(let i= 2; i< fibLength1.length; i++);
+  for(let i= 2; i< fibLength2.length; i++);
+}
```

Comment:
So I guess my first question here is why you didn't make a function named sequenceNums if that was your test function name? 

To bring this up another level you will want to record the output of this function and what you would need to do next to solve for it.